sol/api.sh

```diff
@@ -61,8 +61,8 @@ args=(
 
 # Note: can get into a bad state that requires actually fetching a new snapshot. One such error that indicates this:
 # "...processing for bank 0 must succeed: FailedToLoadEntries(InvalidShredData(Custom(\"could not reconstruct entries\")))"
-if [[ -d /data/sol/ledger ]]; then
-  args+=(--no-snapshot-fetch)
-fi
+#if [[ -d /data/sol/ledger ]]; then
```

Comment:
I think this is probably not exactly right as once the validator is up and running, it should not have to fetch a snapshot if it is stopped and started again. It would probably be good to note in the README that while initially setting up the validators it is probably worthwhile to comment this out so that a half created `/data/sol/ledger` directory doesn't incorrectly trigger this flag and the validator fails to start due to not having a snapshot

Reply:
Ok I changed it, please let me know if this looks right to you.